sqlserver/metadata.csv

```diff
@@ -1,11 +1,24 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
 sqlserver.access.page_splits,gauge,,operation,second,The number of page splits per second.,-1,sql_server,page splits,
+sqlserver.ao.is_primary_replica,gauge,,,,"Returns 1 if the replica is primary, or 0 if it is a secondary replica.",0,sql_server,is primary replica,
```

Comment:
Can you add tags to the documentation? See the metric `sqlserver.files.reads` for examples.